pkg/gui/gui.go

```diff
@@ -124,6 +124,8 @@ type Gui struct {
 	PopupHandler PopupHandler
 
 	IsNewRepo bool
+
+	InitialRepoDir string
```

Comment:
could we rename this to just `InitialDir`, just because it's not necessarily going to be the root directory of the repo